packages/react/src/components/Footer/__stories__/Footer.stories.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { Footer } from '@ibmdotcom/react/es/components/Footer';
+import { Footer } from '@ibmdotcom/react/';
```

Comment:
@photodow you missed the extra slash at the end, shouldn't be `@ibmdotcom/react/`